src/experiments.js

```diff
@@ -124,11 +124,21 @@ export function isExperimentOnAllowUrlOverride(win, experimentId) {
  * @return {boolean}
  */
 function calcExperimentOn(win, experimentId) {
-  const cookieFlag = getExperimentIds(win).indexOf(experimentId) != -1;
+  const experiments = getExperimentIds(win);
+
+  // Disabling cookie flag.
+  const disableFlag = experiments.indexOf('-' + experimentId) != -1;
```

Comment:
This is for a developer to manually opt-out. Basically, "We just launched feature X. Our binary config as X:1". There's no way in this scenario to override X back to 0 manually w/o redeployment. This is useful when you want to check if indeed the launch caused the problem or not.